testing/src/main/java/io/stargate/it/http/restapi/RestApiTestBase.java

```diff
@@ -26,7 +26,9 @@
 import org.junit.jupiter.api.TestInfo;
 
 /** Intermediate IT base class for use by REST API integration tests. */
-abstract class RestApiTestBase extends BaseIntegrationTest {
+abstract class RestApiTestBase extends BaseRestApiTest {
+  // ^^^ 16-Feb-2022, tatu: This odd hierarchy is due to discrepancy between SGv1/SGv2
+  //   and needs to be solved soon.
```

Comment:
Will resolve this next but this PR needed to make CI "green" again until then -- 2 new ITs under `http/restapi` failing until this gets merged.